src/Stack/Build/ConstructPlan.hs

```diff
@@ -747,6 +744,19 @@ addPackageDeps package = do
                                         warn_ "trusting snapshot over Cabal file dependency information"
                                         pure True
                                     else pure False
+                        if allowNewer of
```

Comment:
The CI is currently failing because of the typo in this line.

Reply:
fixed, bad hlint fix :)